README.md

```diff
@@ -20,23 +20,43 @@ Source code annotation tool offers an UI to annotate source code and review thes
 
 ## Installation
 
-## Github OAuth tokens
+## Environment Variables
 
-1. You need an OAuth application on GitHub. See [how to create OAuth applications on GitHub](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/).
+The next sections make use of several environment variables to configure the application. In this table you will find all of them grouped as a quick reference:
 
-In order to be able to use this application while running the tool locally, make sure you add http://127.0.0.1:8080/oauth-callback to the authorization callback URL field.
 
-2. Copy `.env.tpl` to `.env`.
+Variable | Required | Default value | Meaning
+-|-|-|-|-
+`CAT_ENV` | | `production` | Sets the log level. Use `dev` to enable debug log messages.
+`CAT_HOST` | | `0.0.0.0` | IP address to bind the HTTP server
+`CAT_PORT` | | `8080` | Port address to bind the HTTP server
+`CAT_SERVER_URL` | | `<CAT_HOST>:<CAT_PORT>` | TODO
+`CAT_UI_DOMAIN` | | `<CAT_HOST>:<CAT_PORT>` | TODO
+`CAT_DB_CONNECTION` | | `sqlite:///var/code-annotation/internal.db` | Points to the internal application database. Read below for the complete syntax
+`CAT_EXPORTS_PATH` | | `./exports` | Folder where the SQLite files will be created when requested from `http://<your-hostname>/export`
+ | | | |
+`CAT_OAUTH_CLIENT_ID` | YES | - | GitHub application OAuth credentials
+`CAT_OAUTH_CLIENT_SECRET` | YES | - | GitHub application OAuth credentials
+`CAT_OAUTH_RESTRICT_ACCESS` | | - | Application access control based on GitHub groups or teams
+`CAT_OAUTH_RESTRICT_REQUESTER_ACCESS` | | - | User role control based on GitHub groups or teams
+ | | | |
+`CAT_JWT_SIGNING_KEY` | YES | - | Key used to sign JWT (JSON Web Tokens) in the server
 
-3. Retrieve the values for your application's Client ID and Client Secret from the [GitHub Developer Settings page](https://github.com/settings/developers) and add them to the end of the corresponding lines in .env.
+## Github OAuth Tokens
+
+You need an OAuth application on GitHub. See [how to create OAuth applications on GitHub](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/). Make sure the "Authorization callback URL" points to `http://<your-hostname>/oauth-callback`.
+
+Retrieve the values for your application's Client ID and Client Secret from the [GitHub Developer Settings page](https://github.com/settings/developers) and set them to the environment variables `CAT_OAUTH_CLIENT_ID` and `CAT_OAUTH_CLIENT_SECRET`.
 
 ### Docker
 
-docker run --env-file .env --rm -p 8080:8080 srcd/code-annotation
+```bash
+$ docker run --rm -p 8080:8080 srcd/code-annotation
```

Comment:
Here should be example how to set required env variables. At least:
- CAT_OAUTH_CLIENT_ID
- CAT_OAUTH_CLIENT_SECRET

are there any other?

Reply:
Done. Regarding your second comment, I think the reordering of the steps falls into #67, to be done in a future PR.

Reply:
Imo, teaching to the user how to provide a db is out of scope for this PR, and should be done by #67